### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by fwininger**
Test against the last ruby and rails version. Use github action in place of travis.

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-require 'coveralls'
-Coveralls.wear!
+# require 'coveralls'
```
Comment (mschnitzer) [inline test/test_helper.rb:3]: Did you commit this by accident or is this intentional?

**Comment by mschnitzer** [comment]
Thanks for providing this PR. I'm currently running in some issues with the test suite, but those issues seem to be caused by a weak dependency specification (for which I will provide a PR if it turns out to be true) and shouldn't be related to your PR.

I will provide more feedback soon. I left a comment regarding coveralls above.

File: test/test_helper.rb (hunk 1 above)
Comment (fwininger) [reply test/test_helper.rb:3]: This PR is in draft. I have a issue with ruby 3.1 : https://github.com/Cyberwatch/exception_notification/runs/4702998693?check_suite_focus=true

Comment (fwininger) [reply test/test_helper.rb:3]: I have disable coveralls to be able to debug the test suite.

**Comment by fwininger** [comment]
@mschnitzer / @smartinez87 this PR is rebase and finished. I add Github Action, all tests works, i have just a warning with coverall : https://github.com/Cyberwatch/exception_notification/runs/4800684350?check_suite_focus=true

I drop the test against old ruby and rails versions.

**Comment by mschnitzer** [comment]
OK thanks, I will look into this on Mo/Tue.

I think we can release a new version when this PR has been merged.

**Review by mschnitzer — APPROVED** [review]

**Comment by mschnitzer** [comment]
@smartinez87 I consider this change as beneficial to this project. I ran the GitHub actions myself locally and could not spot any issues with them. I requested a review from you.

Given that the last Travis run was one year ago, I would say we can switch to GitHub actions instead.

**Pull request merged**